packages/utilities/src/memoize.ts

```diff
@@ -80,9 +80,16 @@ export function memoize<T extends Function>(
  * @public
  * @param cb - The function to memoize.
  * @param maxCacheSize - Max results to cache. If the cache exceeds this value, it will reset on the next call.
+ * @param doNotCacheIfCallbackResultIsNull - Flag to decide whether to cache callback result if it is null.
```

Comment:
Couple issues with this parameter:

1. Should it skip caching for `null`, `undefined`, or both? It sounds like the specific case you care about involves `undefined`, not `null`. (Either way, the comment should be updated to specify which value(s) are handled specially.)
2. I'd prefer a more concise name like `ignoreUndefinedResult` or `ignoreNullResult` (as appropriate).

Reply:
changed the name to ignoreNullOrUndefinedResult as I want to handle for undefined and null